docs/dashboards/dashboards-with-angular.md

```diff
@@ -26,14 +26,14 @@ Then, import the package and the dedicated plug to connect it to the Dashboards.
 
 ```typescript
 import Highcharts from 'highcharts';
```

Comment:
We might as weel import everything with esm here:
`import Highcharts from 'highcharts/es-modules/masters/highcharts.src.js';`